index.d.ts

```diff
@@ -40,14 +42,15 @@ Map object keys and values into a new object.
 @param source - Source object to copy properties from.
 @param mapper - Mapping function.
 
+@param options? - Any options which can be applied to the operation
```

Comment:
This doesn't add anything for the user. Can be removed.

Reply:
```suggestion

```

For me it adds some more overview that I will find as a developer the option param in this typification, which I didn't expect first. But it is okay for me to remove.